### DISCUSSION TITLE
text-green-500 is not the same between tailwindcss 2 and 3

### DISCUSSION THREAD

**Discussion by Lord-Y** [tailwindlabs/tailwindcss#7442] [Help] [answered]
Hello guys,

I'm not sure if this is something wanted or not but rgba colors between `2.2.19` and `3.0.22`.
On my personal project I saw that for version 2, in the browser console, the result is:
```
.text-green-500 {
    --tw-text-opacity: 1;
    color: rgb(34 197 94 / var(--tw-text-opacity));
}
```
while on version 3 it's:
```
.text-green-500 {
    --tw-text-opacity: 1;
    color: rgba(16,185,129,var(--tw-text-opacity));
}
```

On the documentation [v2](https://v2.tailwindcss.com/docs/customizing-colors#color-palette-reference) and [v3](https://tailwindcss.com/docs/customizing-colors#default-color-palette), the haxadecimal color is the same `#22c55e`

At the end, the color on the version 3 is lighter then the version 2.
Not sure if this is something wanted or not but the color should be more like `#10B981` right?
Any thoughts?

**Comment by NovusOpus** [comment, marked as answer]
This is explained here: [Tailwind CSS Docs - Upgrade Guide - Color palette changes](https://tailwindcss.com/docs/upgrade-guide#color-palette-changes)

**Reply by Lord-Y** [reply]
Thanks @NovusOpus, I missed this part. After changing `text-green-500` to `text-emerald-500`. Everything is okay 👍🏾
